src/rest/rest-router.ts

```diff
@@ -21,6 +22,8 @@ subRouter.use(JwtHandler);
 
 subRouter.use("/oauth", OAuthRouter);
 
+subRouter.use("/installation", GitHubAppsRoute);
```

Comment:
\nit 
Can we name this GitHubAppsRouter, just to make it similar to the other routes?